aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -372,9 +372,13 @@ module aptos_framework::staking_contract {
         emit_event(&mut store.reset_lockup_events, ResetLockupEvent { operator, pool_address });
     }
 
-    /// Convenience function to allow a staker to update the commission percentage paid to the operator.
-    /// TODO: fix the typo in function name. commision -> commission
+    #[deprecated]
```

Comment:
shall we leave a todo here to cleanup later? what's the process to cleanup deprecated entry functions? looks like lots of places would need to be updated if we remove this code.

Reply:
I see. 

@movekevin , @brianrmurphy, what is the implication of the attribute `#[deprecated]`, and what is the proper clean up process?

Reply:
You cannot remove deprecated functions due to backwards compatibility rules.

You first can put `#[deprecated]` and then later make an update to make it abort rather than do any changes.

Reply:
Yeah unfortunately we cannot delete public functions as that'd break backward compat

Reply:
I updated the PR to keep the original one, not adding a new one.